pykafka/cluster.py

```diff
@@ -402,23 +402,24 @@ def get_group_coordinator(self, consumer_group):
         """
         log.info("Attempting to discover offset manager for consumer group '%s'",
                  consumer_group)
-        for i in range(self._max_connection_retries):
+        max_connection_retries = self._max_connection_retries + 5
```

Comment:
If this is a consistent issue, I think I'd prefer to raise the default on `_max_connection_retries` than to disregard its configured value here.